pkg/agent/datamodel/versions.go

```diff
@@ -373,12 +373,14 @@ func getAllKubernetesWindowsSupportedVersionsMap() map[string]bool {
 }
 
 // GetSupportedVersions get supported version list for a certain orchestrator.
-func GetSupportedVersions(orchType string, isUpdate, hasWindows bool) (versions []string, defaultVersion string) {
+func GetSupportedVersions(orchType string, isUpdate, hasWindows bool) ([]string, string) {
```

Comment:
These changes where in order to resolve "nonamereturns". So I defined the named returns in the function